cli/types/cypress.d.ts

```diff
@@ -1908,9 +1908,6 @@ declare namespace Cypress {
      */
     origin(originOrDomain: string, fn: () => void): Chainable
 
-    // TODO: when we find other options to put into the 'data' argument of cy.origin, we may want to overload this type with
-    // a 'data' parameter that contains all data options, including args, and one that contains all data options, excluding args.
-    // This will provide better typings support for whatever args is set to as opposed to an optional undefined
```

Comment:
Adding correct typings is part of the acceptance criteria of any feature work, so I felt this comment wasn't needed in the code. I put it in [the issue for `log: false`](https://github.com/cypress-io/cypress/issues/19885), since that should be where we add more options.